frontend/src/components/UseCategoryDropdown.vue

```diff
@@ -1,39 +1,45 @@
 <template>
-  <select @change="$emit('selected', $event.target.value)" class="UseCategoryDropdown">
-    <option value="">Everything</option>
-    <option :value="category" v-for="category in categories">
-      {{ category }} Uses
-    </option>
-  </select>
+<b-dropdown :text="category" class="UseCategoryDropdown" right>
+  <b-dropdown-item-button class="category-item" @click="onCategorySelected('Everything')">Everything</b-dropdown-item-button>
```

Comment:
`:value` should be an empty "" to disable filtering when "Everything" is selected, and the click event should be `@click="onCategorySelected('')`.